frontend/src/main/web/lib/utils/StringUtils.js

```diff
@@ -0,0 +1,22 @@
+import _ from 'lodash';
+
+var StringUtils = {
+  isEmptyOrNull: function (str) {
+    return _.isUndefined(str) || _.isNull(str) || 0 === str.length;
+  },
+
+  trimLeadingSpace: function (str) {
+    if(this.isEmptyOrNull(str)) {
+      return str;
+    }
+    while (str.substring(0,1) == ' ') {
+      str = str.substring(1, str.length);
```

Comment:
Will only work for normal space character, not other whitespace. Is that intended? Might be better to use a regex like `/^\s*(.*)$/` to just grab everything after leading whitespace more directly without creating intermediate strings in memory.